web/src/bundles/app.ts

```diff
@@ -57,6 +57,7 @@ import "../../styles/widgets.css";
 import "../../styles/print.css";
 import "../../styles/inbox.css";
 import "../../styles/color_picker.css";
+import "../../styles/buttons.css";
```

Comment:
You probably want to import this earlier, as CSS precedence will generally prefer later things as an override at equal precedence, I think.

Probably just before `import "../../styles/components.css";` would be a reasonable placement.